docs/import/import-files/archives.md

```diff
@@ -115,3 +115,29 @@ bin\flux import-archive-files ^
 ```
 {% endtab %}
 {% endtabs %}
+
+## Importing large binary files in archives
+
+If you import an archive that contains large binaries that required use of the `--streaming` option when 
```

Comment:
"that required use of the `--streaming` option when"
Something needs to change here. Not sure what.